domains/yahya.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "hyodu",
+    "email": "github.unflawed727@passinbox.com"
+  },
+  "records": {
+    "CNAME": "https://hyodu.github.io/yahya.github.io/"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.